skimage/measure/_regionprops.py

```diff
@@ -349,7 +349,6 @@ def __eq__(self, other):
 
 def regionprops(label_image, intensity_image=None, cache=True):
     """Measure properties of labeled image regions.
-
```

Comment:
Single blank lines before and between docstring sections are mandatory.
Please, revert this one and the similar changes below.